lib/crowbar/client/request/base.rb

```diff
@@ -60,15 +60,24 @@ def process
           )
 
           case result.code
-          when 500
-            raise InternalServerError,
-              "An internal error occured"
           when 401
             raise NotAuthorizedError,
               "User is not authorized"
           when 403
             raise NotAuthorizedError,
               "User access is forbidden"
+          when 500
+            raise InternalServerError,
```

Comment:
why do you need to implement your own empty HTTP error classes when you have the ones from `net/http`?